# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cups/hotfix-no-avahi.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff -ruN cups-2.4.20.orig/cups/dest.c cups-2.4.20/cups/dest.c
--- cups-2.4.20.orig/cups/dest.c	2026-10-05 17:26:07.000000000 +0200
+++ cups-2.4.20/cups/dest.c	2026-10-11 13:23:26.739356955 +0200
@@ -125,6 +125,8 @@
 {
   int		num_dests;		/* Number of destinations */
   cups_dest_t	*dests;			/* Destinations */
+  int		num_local;		/* Number of local cupsd queues */
+  cups_dest_t	*local_dests;		/* Local cupsd queues */
   char		def_name[1024],		/* Default printer name, if any */
 		*def_instance;		/* Default printer instance, if any */
 } _cups_getdata_t;
